Changes needed to successfully build opensp with the GNU compilers.

Taken from the fix at:
https://sourceforge.net/p/openjade/bugs/133/

This change is already known upstream.

--- OpenSP-1.5.1/include/InternalInputSource.h.orig	2016-09-19 12:19:35.766064218 +0000
+++ OpenSP-1.5.1/include/InternalInputSource.h	2016-09-19 12:20:04.849794945 +0000
@@ -34,7 +34,7 @@
   void pushCharRef(Char ch, const NamedCharRef &);
   Boolean rewind(Messenger &);
   const StringC *contents();
-  InternalInputSource *InternalInputSource::asInternalInputSource();
+  InternalInputSource *asInternalInputSource();
   ~InternalInputSource();
 private:
   InternalInputSource(const InternalInputSource &); // undefined
